### PR TITLE
PR #345 continued

### PR DESCRIPTION
**Pull request by hrshdhgd**
Only 2 changes:
 - [x] Update gitignore and remove large result files from version control
 - [x] `prefix_modifier` column absence handled in `sssom-py` itself and so no code required in this project to address the same.
 - [x] For the pipeline to run smoothly here, this PR needs to be merged first:
   - [x] https://github.com/monarch-initiative/mondo/pull/6454

### CODE REVIEW

**Comment by hrshdhgd** [comment]
I'm still getting 
```
ValueError: ['<https', 'dct', '<http'] are used in the SSSOM mapping set but it does not exist in the prefix map
```

There seem to be IRIs in some sssom file. Do we set `strict = False` while running `msdf.clean_prefixes()`?

**Comment by matentzn** [comment]
All of these need to be added to the prefix map for now. The big problem is that the external prefix map we use in sssom is severely limited, because it only contains canonical prefixes and expansions. If we want this process to to scale, we should do this once and for all by implementing: https://github.com/mapping-commons/sssom-py/issues/395. What do you think?

**Comment by hrshdhgd** [comment]
> All of these need to be added to the prefix map for now. The big problem is that the external prefix map we use in sssom is severely limited, because it only contains canonical prefixes and expansions. If we want this process to to scale, we should do this once and for all by implementing: [mapping-commons/sssom-py#395](https://github.com/mapping-commons/sssom-py/issues/395). What do you think?

Totally agree!

**Comment by hrshdhgd** [comment]
See https://github.com/mapping-commons/sssom-py/pull/396

File: src/scripts/match-mondo-sources-all-lexical.py [hunk 1]
```diff
@@ -131,11 +131,6 @@ def run(input: str, config: str, rules: str, rejects: str, output: str):
     # msdf.df[OBJECT_ID] = msdf.df[OBJECT_ID].apply(
     #     lambda x: iri_to_curie(x) if x.startswith("<http") else x
     # )
-    msdf.df = filter_prefixes(
-        df=msdf.df, filter_prefixes=prefix_of_interest, features=[SUBJECT_ID, OBJECT_ID]
```
[the remark this answers is not in the document]
Comment (hrshdhgd) [reply src/scripts/match-mondo-sources-all-lexical.py:135]: That's what was causing the error below. So fixed!!

**Comment by hrshdhgd** [comment]
`make matches` completes without errors now.

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
Excellent thanks!!

**Pull request merged**